src/cml.js

```diff
@@ -159,19 +159,40 @@ class CML {
     throw new Error(`driver ${driver} unknown!`);
   }
 
+  watermark(driver, scope = 'global') {
+    var tag;
+    var id;
+    switch (scope) {
+      case 'global':
+        return '![CML watermark](https://raw.githubusercontent.com/iterative/cml/master/assets/watermark.svg)';
+      case 'workflow':
```

Comment:
I'd have to revisit my thoughts on this, but I believe this should really be a combo of both these options. I can have two files with the same job id.

or perhaps another scope as an option that tries to be more unique or a user-provided tag.

Reply:
jobId() is reading from `GITHUB_JOB` which is user defined. you can have two separate workflows with the same job id. 

I can have:
```yaml
name: thing1
on: ...
jobs:
  train:
```
and
```yaml
name: thing2
on: ...
jobs:
  train:
```

where id will be `train` for both workflows. thus it should either be `workflow-id + job-id` to be unique or this in addition to what is present. note that workfowId() would return `thing1` and `thing2` respectively here and jobId() would return `train` for both.

Reply:
`job` was a choice for scope name on my part - I've changed it to `run` (see updated PR comment)

Reply:
@tasdomas this change might break Studio. Let me check the studio code.